### DISCUSSION TITLE
Windows 11 ISOs (including ARM) directly from Microsoft, on macOS

### DISCUSSION THREAD

**Discussion by mattieb** [utmapp/UTM#5324] [Show and tell]
This past week, I spent some time getting some basic tools together for getting Windows 11 ISOs directly from Microsoft that run on macOS.

They should be much nicer to use than uupdump, as long as you want a release version of Windows.

So far, only I've used them to get Windows 11 Professional on ARM (which I then activated using my retail license). Feedback and patches are welcome! I've tried to keep them simple so you can understand exactly what they're doing.

This tool will download an ESD (electronic software distribution) from Microsoft. Microsoft has ESDs for all languages, editions, and architectures of Windows 11. https://github.com/mattieb/download-windows-esd

This tool, along with [wimlib](https://wimlib.net), will take an ESD and convert it into an ISO image that UTM can use. https://github.com/mattieb/windows-esd-to-iso

I hope they're helpful!

**Comment by dmuiX** [comment]
Thanks a lot. Seems to work pretty good so far 🗡️
